app/models/manageiq/providers/vmware/infra_manager.rb

```diff
@@ -399,6 +399,11 @@ def vm_acquire_mks_ticket(vm, options = {})
     end
     alias_method :vm_remote_console_mks_acquire_ticket, :vm_acquire_mks_ticket
 
+    def vm_acquire_ticket(vm, options = {})
+      invoke_vim_ws(:acquireTicket, vm, options[:user_event], options[:ticket_type])
+    end
+    alias_method :vm_remote_console_acquire_ticket, :vm_acquire_ticket
```

Comment:
So the alias for mks is used here https://github.com/ManageIQ/manageiq-providers-vmware/blob/master/app/models/manageiq/providers/vmware/infra_manager/vm/remote_console.rb#L44
Is there a reason to use the alias and not the main method?